### PR TITLE
add live view breadcrumb

### PR DESCRIPTION
**Pull request by leangseu-edx**
Ticket: [AU-325](https://openedx.atlassian.net/browse/AU-325)

Adding breadcrumb with proper link. 

image

### CODE REVIEW

**Comment by muselesscreator** [comment]
Looks good, but missing a couple of pieces:
* Please also add courseId to the mock data in `data/services/lms/fakeData/course'
 - I recommend `course-v1:Foo+TT101+R0` (The course ID for Nathan's time travel course)
* Also, this still needs the link to the ORA URL next to the title.

File: src/data/constants/app.js [hunk 1]
```diff
@@ -2,3 +2,4 @@ import { getConfig } from '@edx/frontend-platform';
 
 // eslint-disable-next-line import/prefer-default-export
 export const routePath = `${getConfig().PUBLIC_PATH}:courseId`;
+export const locationId = window.location.pathname.slice(1)
```
Comment (leangseu-edx) [inline src/data/constants/app.js:5]: should this be function in some helper instead?

Comment (muselesscreator) [reply src/data/constants/app.js:5]: currently, is only used in one place.  If we at some point need to do it more than once, we should make a helper.

**Review by muselesscreator — APPROVED** [review]
LGTM

**Pull request merged**